Makefile

```diff
@@ -61,7 +61,7 @@ lint:
 ### gofmt:                Format all go codes
 .PHONY: gofmt
```

Comment:
Can we use a more consistent name, such as `update-gofmt`?

Reply:
Thanks, Done in [aac3c65](https://github.com/apache/apisix-ingress-controller/pull/885/commits/aac3c6529b1164267f8f27f7cee8a357839b2496) commit.